nupic/regions/SPRegion.py

```diff
@@ -470,50 +452,16 @@ def _allocateSpatialFDR(self, rfInput):
                      for name in self._spatialArgNames)
     
     # Instantiate the spatial pooler class.
-    if ( (self._FDRCSpatialClass == CPPSpatialPooler) or
-         (self._FDRCSpatialClass == PYSpatialPooler) ):
+    if ( (self.SpatialClass == CPPSpatialPooler) or
+         (self.SpatialClass == PYSpatialPooler) ):
       
       autoArgs['columnDimensions'] = [self.columnCount]
       autoArgs['inputDimensions'] = [self.inputWidth]
       autoArgs['potentialRadius'] = self.inputWidth
     
-      self._sfdr = self._FDRCSpatialClass(
+      self._sfdr = self.SpatialClass(
         **autoArgs
       )
-      
-    else:
-      # Backward compatibility
-      autoArgs.pop('coincidencesShape')
-      autoArgs.pop('inputShape')
-      autoArgs.pop('inputBorder')
-      autoArgs.pop('coincInputRadius')
-      autoArgs.pop('cloneMap')
-      autoArgs.pop('numCloneMasters')
```

Comment:
see my comment above, these are the params to document for compatibility reasons.